### PR TITLE
Fix AESEncryptField for python3

### PR DESCRIPTION
**Pull request by ningirsu**
In python3 you can't create a bytes string with bytes(' ') because you need to specify an encoding.

```
2017-05-05 22:34:43,025 ERROR      File "/home/moutix/.virtualenvs/stepmania-server/lib/python3.5/site-packages/playhouse/fields.py", line 320, in get_cipher
    key = key + bytes(' ') * (32 - len(key))

2017-05-05 22:34:43,025 ERROR    Uncaught exception: string argument without an encoding
```

### CODE REVIEW

**Comment by ningirsu** [comment]
Related to: https://github.com/coleifer/peewee/issues/1262

**Comment by ningirsu** [comment]
The tests for the playhouse extra_fields are not run automatically. Is there a particular reason?

File: playhouse/fields.py [hunk 1]
```diff
@@ -317,7 +317,7 @@ def __init__(self, key, *args, **kwargs):
         def get_cipher(self, key, iv):
             if len(key) > 32:
                 raise ValueError('Key length cannot exceed 32 bytes.')
-            key = key + bytes(' ') * (32 - len(key))
+            key = key + b' ' * (32 - len(key))
```
Comment (coleifer) [inline playhouse/fields.py]: Why was this necessary?

**Comment by coleifer** [comment]
Will all those byte literals be compatible with Python 2.6?

**Comment by coleifer** [comment]
See comments on #1262 and #1264
